src/components/EditorMarkdownIt.vue

```diff
@@ -52,6 +53,40 @@ export default {
 		onUpdate() {
 			this.html = this.md.render(this.value)
 		},
+
+		onChange() {
+			const markdown = this.value
+			const updateFunction = this.updateMarkdown
+			const items = document.getElementsByClassName('task-list-item')
+			for (let i = 0; i < items.length; ++i) {
+				items[i].addEventListener('click', function() {
+					const wasChecked = !this.children[0].checked
+
+					let idOfCheckbox = 0
+					const markdownLines = markdown.split('\n')
+					markdownLines.forEach((line, i) => {
+						if (line.trim().startsWith('- [x]') || line.trim().startsWith('- [ ]')) {
```

Comment:
Does this respects every possible syntax? E.g. capital `X` vs. small `x`, additional spaces after the dash `-`, non-breaking space instead of normal space. See also https://github.com/linsir/markdown-it-task-checkbox/blob/master/index.js#L121

Reply:
I have switched to the regex aswell to make sure it works